an-apluss/job_listing_API
# Implement the functionality to sign in a user and generate token

#### What does this PR do?
- Implement `POST  /api/auth/signin` endpoint functionality

#### Description of Task to be completed?
- include a template for the environment variable
- implement jwt-auth to generate token after login
- add route pattern for user sign in
- add logic to sign in a user
- update .env.sample file with jwt private key

#### How should this be manually tested?
- Clone the repository using `git clone https://github.com/an-apluss/job_listing_API.git` . in your terminal
- run `php -S localhost:8000 -t public` on your terminal to start the server
- make a `POST`  request to the endpoint `localhost:8000/api/auth/signup` with the body request on postman 
- You should see the response template like below
````
{
    "access_token": "Bearer eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJp..",
    "expires_in": 3600,
    "success": true
 }
````
